.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -35,26 +35,27 @@ jobs:
           PR_NUMBER: ${{ github.event.number }}
         name: "Set milestone on related issues in brave-browser"
         run: |
-          URL_REGEX='(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'
+          BB_ISSUE_URL_REGEX='(http(s)?:\/\/)?github.com/brave/brave-browser/issues/\d+'
           UPLIFT_PR_REGEX='uplift\s+(?:of)?:?\s+\K(#\d+)'
-          RESOLVE_ISSUE_REGEX="(?:resolv|fix|clos)(?:e|es|ed)?:?\s+\K(${URL_REGEX})"
+          RESOLVE_ISSUE_REGEX="(?:resolv|fix|clos)(?:e|es|ed)?:?\s+\K(${BB_ISSUE_URL_REGEX})"
+          verlte() { printf '%s\n%s' "$1" "$2" | sort -C -V; }
 
-          uplift_pr_body=$(gh pr view "$PR_NUMBER" -R "$REPO_NAME" --json body --jq '.body')
+          pr_body=$(gh pr view "${PR_NUMBER}" -R "${REPO_NAME}" --json body --jq .body)
 
-          issues=$(echo "${uplift_pr_body}" | grep -ioP "${RESOLVE_ISSUE_REGEX}" || :)
-          if [[ -z ${issues} ]]; then
+          mapfile -t issues < <(echo "${pr_body}" | grep -ioP "${RESOLVE_ISSUE_REGEX}" || :)
+          if [[ -z "${issues[*]}" ]]; then
```

Comment:
So if we have an issue in the PR, we won't collect issues from uplifts and add them to the list. Is this what we want?

Reply:
I've updated to merge issues (from PR, and from uplifted PRs)